headless-server-commerce-parent/pom.xml

```diff
@@ -396,12 +393,10 @@
             <phase>validate</phase>
             <configuration>
               <rules>
-                <myCustomRule implementation="com.coremedia.maven.enforcer.ModularOneRepoEnforcerRule">
+                <oneRepoEnforcerRule />
```

Comment:
I couldn't find out which version of the maven-enforcer-plugin you are using.
This new format for the rules must of course match with the plugin version. (in the cms we use 3.4.1).
Other than that it looks good to me.

Reply:
maven-enforcer-plugin 3.5.0 is used, see build output: https://github.com/CoreMedia/coremedia-headless-commerce/actions/runs/14835445885/job/41645584518?pr=316#step:7:70

The plugin version is inherited/aggregated from the `spring-boot-dependencies` via `spring-boot-starter-parent`, see `<parent/>` node.